Assets/PatchKit/UnityEditorExtension/Editor/UI/CreateAppScreen.cs

```diff
@@ -69,22 +70,31 @@ public override void Draw()
         GUILayout.EndHorizontal();
         EditorGUILayout.BeginHorizontal();
         {
-            GUILayout.Label(new GUIContent("Target platform:     " + EditorUserBuildSettings.activeBuildTarget,"PatchKit application target platform is unambiguous with current active project build platform."));
+            GUILayout.Label(new GUIContent("Target platform:     " + EditorUserBuildSettings.activeBuildTarget, 
+                                           "PatchKit application target platform is unambiguous with current active project build platform."));
             GUILayout.FlexibleSpace();
-            if (GUILayout.Button(
-                new GUIContent(
-                    "Switch Platform",
-                    "Change PatchKit application target platform."),
-                GUILayout.Width(110)))
+        }
+        EditorGUILayout.EndHorizontal();
+        EditorGUILayout.Separator();
+        using (Style.ColorifyBackground(Color.blue))
+        {
+            var style = new GUIStyle(GUI.skin.label);
+            style.normal.textColor = Color.blue;
+           
+            if (GUILayout.Button("Need to change platform?", style, GUILayout.ExpandWidth(false)))
             {
-                if (EditorUtility.DisplayDialog("Warning", "To change application platform, you need to switch the project platform. \n\nRemember, PatchKit support only: Windows, Mac, Linux platforms.",
-                                                "Change the project platform", "Cancel"))
-                {
-                    Dispatch(() => SwitchPlatform());
-                }
+                EditorUtility.DisplayDialog(
+                    "Need to change platform?",
+                    "PatchKit application target platform is unambiguous with current active project build platform.\n\n" +

```

Comment:
I think that word `unambiguous` might be hard to understand by most of the developers, that aren't mainly native English spekears. 

I think that "While uploading application from Unity, PatchKit target platform is determined by current active project build platform." would be better.